preview-worker/Dockerfile

```diff
@@ -0,0 +1,75 @@
+FROM alpine:3.4
+
+MAINTAINER Simon Forsman <simon@urb-it.com>
+
+ENV SUPERVISOR_VERSION=3.3.1
+ENV GOPATH /go
+ENV GO15VENDOREXPERIMENT 1
+ENV KUBE_VERSION="v1.4.6"
+ENV HELM_VERSION="v2.0.2"
+ENV HELM_FILENAME="helm-${HELM_VERSION}-linux-amd64.tar.gz"
+
+ADD ./worker.conf /etc/supervisord.conf
+
+# Install dependencies
+RUN echo http://dl-4.alpinelinux.org/alpine/edge/testing \
+>> /etc/apk/repositories \
+&& echo http://dl-4.alpinelinux.org/alpine/edge/main \
+>> /etc/apk/repositories \
+&& echo http://dl-4.alpinelinux.org/alpine/edge/community \
+>> /etc/apk/repositories \
+&& apk --no-cache --update add \
+   gcc \
+   curl \
+   perl \
+   openssl-dev \
+   tar \
+   unzip \
+   g++ \
+   git \
+   make \
+   cmake \
+   openssl \
+   go \
+   ca-certificates \
+   luajit \
+   py-pip \
+&& update-ca-certificates
+
+# kubectl and helm
+RUN curl -L https://storage.googleapis.com/kubernetes-release/release/${KUBE_VERSION}/bin/linux/amd64/kubectl -o /usr/local/bin/kubectl \
+    && chmod +x /usr/local/bin/kubectl \
+    && curl -L http://storage.googleapis.com/kubernetes-helm/${HELM_FILENAME} -o /tmp/${HELM_FILENAME} \
+    && tar -zxvf /tmp/${HELM_FILENAME} -C /tmp \
+    && mv /tmp/linux-amd64/helm /usr/local/bin/helm \
+    && chmod +x /usr/local/bin/helm && helm init -c && helm repo add urbit http://charts.urb-it.io
+
+# CLI53
+RUN go get github.com/barnybug/cli53 ;\
+    cd $GOPATH/src/github.com/barnybug/cli53 ;\
+    make install
+
+# Python packages
+RUN pip install \
+    supervisor==$SUPERVISOR_VERSION \
+    supervisor-stdout \
+    redis \
+    enum34
+
+# Cleanup
+RUN apk --no-cache del \
```

Comment:
It is possible, i'd prefer to keep each set of changes (system packages, kube-stuff, python-stuff, etc) separated but i can merge it all to a single big run statement.